src/gfx/context.cpp

```diff
@@ -79,10 +137,10 @@ struct ContextMainOutput {
 
 #if GFX_EMSCRIPTEN
     auto callback = [](double time, void *userData) -> EM_BOOL {
-      spdlog::debug("Animation frame callback {}", time);
+      getLogger()->debug("Animation frame callback {}", time);
       return true;
     };
-    spdlog::debug("Animation frame queued");
+    getLogger()->debug("Animation frame queued");
     emscripten_request_animation_frame(callback, this);
     emscripten_sleep(0);
```

Comment:
So here, if you have `CBContext` it can definitely be replaced with `CB_SUSPEND` and it will allow the node to process other chains too.
`emscripten_sleep` uses the same async/await facilities internally it's an hack.

Reply:
If it's not critical code just add a note btw for later review.

Reply:
Forgot it was used here too, technically not necessary, only in standalone sample code.
Removed it here since the chainblocks main loop should take care of it already if it calls sleep with asyncify